packages/components/src/Button/Button.stories.mdx

```diff
@@ -3,22 +3,20 @@ import { Button } from './Button';
 
 <Meta title="Components/Button" />
 
-# Button-Component
+# Button
 
 ## Description
 
 With the Button component you can render a `button` element.
-By adding the `themeSection` prop you can change to another themeSection. `button` is the default section.
-The variant in your section can be added with the variant prop. E.g. `variant="primary.large"` or `variant="secondary.small"`.
-Specific css style prop can also be added.
+The style variant and size of the button can be added with the variant prop and is composed like `variant.size`.
+Custom CSS styles can be added with the CSS prop.
 
 ## Properties
 
-| Property       | Type        | Default    |
-| :------------- | :---------- | :--------- |
-| `themeSection` | `string`    | `'button'` |
-| `variant`      | `string`    | `''`       |
-| `css`          | `css props` |            |
```

Comment:
Why you've deleted the css prop?

Reply:
they should not be exposed by our components ;)

Reply:
because it's a "default" property for all components and specified in the Box; I thought it's redundant as we don't list all Button Attributes either, but we can leave it if you want to :)

Reply:
It's ok for me. I think we will see how good the users understand the docs :)